cli/internal/cloudcmd/validators.go

```diff
@@ -24,45 +26,77 @@ func NewValidator(cmd *cobra.Command, config config.AttestationCfg, log debugLog
 	return choose.Validator(config, warnLogger{cmd: cmd, log: log})
 }
 
-// UpdateInitPCRs sets the owner and cluster PCR values.
-func UpdateInitPCRs(config config.AttestationCfg, ownerID, clusterID string) error {
+// UpdateInitMeasurements sets the owner and cluster measurement values.
+func UpdateInitMeasurements(config config.AttestationCfg, ownerID, clusterID string) error {
 	m := config.GetMeasurements()
-	if err := updatePCR(m, uint32(measurements.PCRIndexOwnerID), ownerID); err != nil {
+
+	switch config.GetVariant() {
+	case variant.AWSNitroTPM{}, variant.AzureTrustedLaunch{}, variant.AzureSEVSNP{}, variant.GCPSEVES{}, variant.QEMUVTPM{}:
+		if err := updateMeasurementTPM(m, uint32(measurements.PCRIndexOwnerID), ownerID); err != nil {
+			return err
+		}
+		return updateMeasurementTPM(m, uint32(measurements.PCRIndexClusterID), clusterID)
+	case variant.QEMUTDX{}:
+		return updateMeasurementTDX(m, uint32(measurements.TDXIndexClusterID), clusterID)
```

Comment:
@daniel-weisse

Reply:
We also dont actually measure `ownerID` in the TPM case, since the value is always nil. (We have an rfc on how it could look like if we decide to re-implement the `ownerID` measuring)

Since afaik, measuring two values at runtime would require extra logic for TDX, we decided to not implement that path at the current moment, and left it open to the future.
I agree that a comment for this could be useful here.

Reply:
Could you phrase this as a comment? I can add it.

Reply:
```suggestion
		// Measuring ownerID is currently not implemented for Constellation
		// Since adding support for measuring ownerID to TDX would require additional code changes,
		// the current implementation does not support it, but can be changed if we decide to add support in the future
		return updateMeasurementTDX(m, uint32(measurements.TDXIndexClusterID), clusterID)
```